tests/dynamic_friction.prm

```diff
@@ -47,20 +35,26 @@ subsection Mesh deformation
 end
 
 subsection Material model
-
-  set Model name = dynamic friction
-  subsection Dynamic Friction
+  set Model name = visco plastic
+  subsection Visco Plastic
     set Densities         = 2800
     set Viscosity averaging scheme = harmonic
-    subsection Viscosities
-       set Minimum viscosity = 1e19
-       set Maximum viscosity = 1e25
-       set Reference strain rate = 1e-15
-       set Background Viscosities = 1.e25, 1.e20
-       set Coefficients of static friction = 0.5547 , 0.5
-       set Coefficients of dynamic friction = 0.6 , 0.5
-       set Cohesions = 20e6
-    end
+    set Minimum viscosity = 1e19
+    set Maximum viscosity = 1e25
+    set Reference strain rate = 1e-15
+
+    # Dislocation creep parameters for
+    set Prefactors for dislocation creep          =      1e-20, 1e-25
+    set Stress exponents for dislocation creep    =      1.0
+    set Activation energies for dislocation creep =      0.0
+    set Activation volumes for dislocation creep  =      0.0
+
+  # Plasticity parameters
+  set Friction dependence mechanism       = dynamic friction
+  set Dynamic characteristic strain rate  = 1e-15
+    set Angles of internal friction         = 29, 26.6
+    set Dynamic angles of internal friction = 31, 26.6
```

Comment:
Maybe the roundoff of the friction angles also plays a role in the differences you see with the new mm?

Reply:
thought about that too. Will try it again

Reply:
nope, setting the friction angles with less roundoff unfortunately does not make the density curves fit.

Reply:
This might be the cause of the difference in density:
In line 142 of dynamic_friction.cc a default thermal expansivity is set to 4e-5. But for visco_plastic, the default is 0. So please try by setting the thermal expansivity from the input file to 4e-5.

Reply:
yes, density is fine now